pkg/registry/client_http_test.go

```diff
@@ -63,6 +65,19 @@ func (suite *HTTPRegistryClientTestSuite) Test_4_ManInTheMiddle() {
 	suite.True(errdefs.IsFailedPrecondition(err))
 }
 
+func (suite *HTTPRegistryClientTestSuite) SetupTest() {
+	suite.VariableThatShouldStartAtFive = 5
+}
+
+func (suite *HTTPRegistryClientTestSuite) TestExample() {
+	if suite.Equal(5, suite.VariableThatShouldStartAtFive) {
+		assert.Equal(suite.T(), 5, suite.VariableThatShouldStartAtFive)
```

Comment:
Why is this assert package being added when testify can already do what's being used?